install

```diff
@@ -390,6 +390,8 @@ Dir.chdir HOMEBREW_REPOSITORY do
   end
 end
 
+system "#{HOMEBREW_PREFIX}/bin/brew", "tap", "linuxbrew/xorg" unless mac?
```

Comment:
This works if `git` is already available or `$HOMEBREW_PREFIX/bin` is in the PATH.

(In the latter case, it will get installed automatically. But if the PATH Is not set properly, you get this paradoxical output:

```
🍺  /home/bob/.linuxbrew/Cellar/git/2.14.1: 1,408 files, 42.5MB, built in 35 seconds
Error: Git must be installed and in your PATH!
==> Installing git
Warning: git 2.14.1 is already installed
Error: Git is unavailable
Failed during: /home/bob/.linuxbrew/bin/brew tap linuxbrew/xorg
```